### DISCUSSION TITLE
DDP with shared file system

### DISCUSSION THREAD

**Discussion by YuShen1116** [Lightning-AI/pytorch-lightning#8496] [DDP / multi-GPU / multi-node] [answered]
Is it possible to use shared filesystem for DDP init_group in pytorch lighting? If so how what should I do to the Trainer?

Thanks!

**Comment by carmocca** [comment, marked as answer]
I'm not quite sure about what you want to do, but if it's about customizing `DDP`, you can do the following:

```python
from pytorch_lightning.plugins import DDPPlugin

class MyCustomDDP(DDPPlugin):
    ...

trainer = Trainer(plugins=[MyCustomDDP()])
```
